gateway/ln-gateway/src/lib.rs

```diff
@@ -532,6 +541,19 @@ impl Gateway {
             return ReceivePaymentStreamAction::ImmediatelyRetry;
         }
 
+        /*
+        TODO: Uncomment this
+        if synced_to_chain {
```

Comment:
This slows down devimint. I think I will make it optional via an environment variable.

Reply:
I'm guessing this slows down devimint because `wait_for_chain_sync()` uses `backoff_util::background_backoff()`. Would it maybe make more sense to have an environment variable that makes the background backoff quicker rather than dealing with the consequences of it ad-hoc?

Reply:
I don't think so. In production, it will take awhile to sync so its likely not a big deal. I think we want an environment variable to skip it anyway, there might be some scenarios where it is not desirable to wait for the chain to sync

Reply:
The maximum is 60s which is plenty fast for production